app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= stylesheet_link_tag "style.css", "data-turbo-track": "reload" %>
+<h2>Sign up</h2>
```

Comment:
- [ ] Kindly improve the layout of the sign up screen so that the form elements are larger and stretch across the page based on the required wireframe. The image below is what you have now.

  ![image](https://github.com/c00p75/blog-rails/assets/81468675/286ce548-2e84-45b8-9d54-0cee82e7d7af)